src/ImageSharp/IImage.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using SixLabors.Primitives;
+
+namespace SixLabors.ImageSharp
+{
+    /// <summary>
+    /// Encapsulates the properties and methods that describe an image.
+    /// </summary>
+    public interface IImage : IImageInfo, IDisposable
+    {
+        /// <summary>
+        /// Gets the size of the image.
+        /// </summary>
+        /// <returns>The <see cref="Size"/></returns>
+        Size Size();
+
+        /// <summary>
+        /// Gets the bounds of the image.
```

Comment:
`Bounds()`  could be an extension method on `IImageInfo`.

Reply:
I'm happy to move them there. 👍